### PR TITLE
docs(Commando): Update installation instructions

### PR DESCRIPTION
**Pull request by iShibi**
**Please describe the changes this PR makes and why it should be merged:**

Closes: #596

### CODE REVIEW

File: guide/commando/README.md [hunk 1]
```diff
@@ -4,13 +4,10 @@ pageTheme: blue
...
 
-When you got your first bot up and running with Discord.js, you should've installed Discord.js using npm, Node.js' Package Manager. The same applies to Commando, which must be separately installed. You can do this in one of two ways:
-
-If using Discord.js v11: `npm install discord.js-commando`  
-If using Discord.js v12: `npm install discordjs/Commando`
+When you got your first bot up and running with Discord.js, you should've installed Discord.js using npm, Node.js' Package Manager. The same applies to Commando, which must be separately installed. To install Commando, run `npm install discord.js-commando` in the terminal.
 
 ::: warning
-You need at least Node.js version <branch version="11.x" inline>8.0.0</branch><branch version="12.x" inline>12.0</branch> to use Commando. The main branch will also require you to install [Git](https://git-scm.com/downloads).
+You need at least Node.js version <branch version="11.x" inline>8.0.0</branch><branch version="12.x" inline>12.0</branch> to use Commando.
```
Comment (kyranet) [inline guide/commando/README.md:10]: `11.x` is not longer supported in Commando, so there's no point on keeping the branch.
```suggestion
You need at least Node.js version 12.0 to use Commando.
```

Comment (kyranet) [reply guide/commando/README.md:10]: I also thought of doing this instead:
```suggestion
You need <branch version="11.x" inline>to upgrade discord.js to 12.x and </branch>at least Node.js version 12.0 to use Commando.
```

So when it's in 11.x, it shows:

```markdown
:::warning
You need to upgrade discord.js to 12.x and at least Node.js version 12.0 to use Commando.
:::
```

And when in 12.x:

```markdown
:::warning
You need at least Node.js version 12.0 to use Commando.
:::
```

**Review by kyranet — CHANGES_REQUESTED** [review]

File: guide/commando/README.md (hunk 1 above)
Comment (iShibi) [reply guide/commando/README.md:10]: There are a lot of places in the guide where this [condition](https://discordjs.guide/preparations/#installing-on-linux) arises. Wouldn't it be inconsistent to add this change only here? I feel the popup is enough to tell the user that v11 isn't supported anymore.

Comment (kyranet) [reply guide/commando/README.md:10]: I honestly don't know. cc: @discordjs/guides

Comment (almostSouji) [reply guide/commando/README.md:10]: There's still a djs-v11 branch for commando, or am I missing something?
We generally recommend updating to v12, however, keeping the v11 guide intact as far as possible is preferable.

Comment (iShibi) [reply guide/commando/README.md:10]: I assume this is resolved.

Comment (almostSouji) [reply guide/commando/README.md:10]: Well, __if__ that branch actually works with v11, the install instructions still differ and I suppose v11 requires git then. Should that not be the case and commando not function with v11 whatsoever the commando guide itself would need the branching tags and display that fact for 11

**Review by dragonfire535 — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

**Pull request merged**